src/hkml_view_mails.py

```diff
@@ -334,9 +335,11 @@ def do_export(data, answer, selection):
     try:
         answer = int(answer)
     except:
-        print('wrong input.  Return to hkml')
         time.sleep(1)
```

Comment:
We were having `time.sleep()` here, because users will not have time to read the error messages printed above, before going back to curses mode via `shell_mode_end()`.  Now `shell_mode_end()` gives users a time to read the outputs on terminal before going back to curses mode, so this `time.sleep()` can be removed.  Obviously not a blocker of tis PR.